src/components/addToFavorite/addToFavorite.scss

```diff
@@ -0,0 +1,21 @@
+@import "../../variabls.scss";
+
+.like-button {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-color: transparent;
+  border: 1px solid $icons;
+  cursor: pointer;
+  width: 100%;
+  max-width: 48px;
+}
+  
+.like-button:hover {
+  border: 1px solid $primary;
+}
+
+.hearth {
+  width: 16px !important;
```

Comment:
try to avoid important it is bad practice

Reply:
i know but without it doesn't work

Reply:
Just add an own class to your product image so your heart-icon will have own class as well
<img width="1294" alt="Screenshot 2023-03-28 at 19 39 52" src="https://user-images.githubusercontent.com/117587463/228309542-63c71dd3-073b-4cc6-9ae1-ccdbf50e5077.png">